src/openms/include/OpenMS/KERNEL/Peak1DT.h

```diff
@@ -0,0 +1,396 @@
+// SOA and AOS tests on god bolt
```

Comment:
I would probably call that file differently. Something like MSSpectrumAoSWrapper.h